### DISCUSSION TITLE
How to run the project

### DISCUSSION THREAD

**Discussion by megagrzesiek** [LivingWithHippos/unchained-android#324] [General]
Hello, I'm trying to try this project out but I am completely confused, I've cloned the repository but I am unable to find instructions in it on how to run the project. What am I missing? Can anyone help??

**Comment by LivingWithHippos** [comment]
This project was built with [Android studio](https://developer.android.com/studio) (you may need to download the beta version instead of the stable one, can't remember right now), once you've installed it you can do "open project" and select the "app" subfolder in the unchained directory you have cloned. Android studio should automatically import everything and download the gradle dependencies.
